flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why?

Reply:
I haven't tested on ARM yet, so to avoid any issues I've restricted to `x86_64` only for now.

Reply:
nobody will test it if you don't build it.

Reply:
restrictions removed :)